diesel_cli/src/pretty_printing.rs

```diff
@@ -82,7 +82,7 @@ pub fn format_schema(schema: &str) -> Result<String, FmtError> {
         }
     }
 
-    Ok(out.replace("\t", "    "))
+    Ok(out.replace("\t", "    ").replace("table!", "\ntable!").trim().to_string())
```

Comment:
I want to completely rewrite this at some point. I seriously doubt that this function should be operating on token streams at all. But for now I just wanted to get the minimum possible change for the user visible changes.